cmd/werf/common/common.go

```diff
@@ -99,6 +99,11 @@ func SetupConfigTemplatesDir(cmdData *CmdData, cmd *cobra.Command) {
 	cmd.Flags().StringVarP(cmdData.ConfigTemplatesDir, "config-templates-dir", "", os.Getenv("WERF_CONFIG_TEMPLATES_DIR"), `Custom configuration templates directory (default $WERF_CONFIG_TEMPLATES_DIR or .werf in working directory)`)
 }
 
+func SetupConfigRenderDir(cmdData *CmdData, cmd *cobra.Command) {
+	cmdData.ConfigRenderDir = new(string)
+	cmd.Flags().StringVarP(cmdData.ConfigRenderDir, "config-render-dir", "", os.Getenv("WERF_CONFIG_RENDER_DIR"), `Custom directory for storing rendered configuration files (default $WERF_CONFIG_RENDER_DIR or .werf in working directory)`)
```

Comment:
- We should give the user the option to set a custom path (not a directory).
- `.werf` is a default folder for user templates ([rel](https://werf.io/docs/v2/usage/project_configuration/werf_yaml_template_engine.html#template-directory)).
- By default rendered werf.yaml stored somewhere in the tmp directory (`--tmp-dir`), but it may not be described here.